### PR TITLE
test: add clearTimeout and clearInterval

### PR DESCRIPTION
**Pull request by miracles1919**
[[中文版模板 / Chinese template](https://github.com/alibaba/hooks/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
目前会 failed，等 #1695 合了就好了

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     add clearTimeout and clearInterval      |
| 🇨🇳 Chinese |    add clearTimeout and clearInterval       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: packages/hooks/src/useInterval/__tests__/index.test.ts [hunk 1]
```diff
@@ -55,5 +55,6 @@ describe('useInterval', () => {
     jest.advanceTimersByTime(70);
     // not to be called
     expect(callback).toHaveBeenCalledTimes(0);
+    expect(clearInterval).toHaveBeenCalledTimes(1);
```
Comment (hchlq) [inline packages/hooks/src/useInterval/__tests__/index.test.ts:58]: 这里建议不要直接使用 `clearInterval` 具体可以看下 https://jestjs.io/docs/timer-mocks

**Review by hchlq — CHANGES_REQUESTED** [review]

**Review by hchlq — DISMISSED** [review]

**Comment by hchlq** [comment]
这个要等到 #1695 合并之后才合，否则 test 测试用例走不过哈

File: packages/hooks/src/useInterval/__tests__/index.test.ts (hunk 1 above)
Comment (miracles1919) [reply packages/hooks/src/useInterval/__tests__/index.test.ts:58]: 这里其实用的就是 mock 的 `clearInterval`，因为最前面用了 `jest.useFakeTimers`

Comment (miracles1919) [reply packages/hooks/src/useInterval/__tests__/index.test.ts:58]:
```js
console.log(clearInterval);
jest.useRealTimers();
console.log(clearInterval);
```

image

**Review by hchlq — APPROVED** [review]

**Review by crazylxr — APPROVED** [review]

**Pull request merged**